api/urls.py

```diff
@@ -1,18 +1,62 @@
-from django.conf.urls import url, include
-from rest_framework.routers import DefaultRouter
-from api import views
-from rest_framework.schemas import get_schema_view
-from rest_framework_swagger.views import get_swagger_view
+from django.urls import path
+from rest_framework.urlpatterns import format_suffix_patterns
 
-
-router = DefaultRouter()
-router.register(r'songs', views.SongViewSet)
-
-
-#schema view
-schema_view = get_swagger_view(title='Dead Songs')
+from . import views
 
 urlpatterns = [
```

Comment:
obviously all of these neighborhoods specific urls should be taken out but I left this file unchanged for context because it seemed like there was a difference in how the team repo vs the exemplar repo dealt with urls and swagger.  (see also dead_songs/urls.py)